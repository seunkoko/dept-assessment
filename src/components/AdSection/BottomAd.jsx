// eslint-disable-next-line
import React from 'react';

import ImageContainer from '../ImageContainer/ImageContainer';
import AdContent from './AdContent';

import './Ad.scss';

import { imageData, bottomFirstNews, bottomSecondNews } from '../../utils/adData';

const BottomAd = (props) => {
	return (
    <>
      <div className="bottom-ad-style">
        <div className="news-content">
          <AdContent
            firstNews={bottomFirstNews}
            secondNews={bottomSecondNews}
          />
        </div>

        <div className="image-ad hide-on-phone">
          <ImageContainer
            data={
              {
                ...imageData,
                id: 'bottomAdDImage'
              }
            }
          />
        </div>
      </div>
    </>
	);
};

export default BottomAd;
