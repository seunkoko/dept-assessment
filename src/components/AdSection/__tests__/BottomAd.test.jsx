import React from 'react';

import { shallow } from 'enzyme';

import BottomAd from '../BottomAd';

describe('BottomAd', () => {
	const component = shallow(<BottomAd />);

	it('should render BottomAd component', () => {
		expect(component).toMatchSnapshot();
  });
  
});