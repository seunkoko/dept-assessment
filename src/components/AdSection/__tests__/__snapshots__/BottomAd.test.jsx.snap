// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomAd should render BottomAd component 1`] = `
<Fragment>
  <div
    className="bottom-ad-style"
  >
    <div
      className="news-content"
    >
      <AdContent
        firstNews={
          Object {
            "id": "bottomFirstNews",
            "moreDescription": "",
            "news": "Enhancing customer experience for personalised holiday planning using real time data",
            "title": "BUTLINS",
          }
        }
        secondNews={
          Object {
            "id": "bottomSecondNews",
            "moreDescription": "",
            "news": "Predicting booking behavior for holidays with smart data, voice search and machine learning",
            "title": "VACANSESELECT",
          }
        }
      />
    </div>
    <div
      className="image-ad hide-on-phone"
    >
      <ImageContainer
        data={
          Object {
            "description": "Rethinking the online ecosystem",
            "id": "bottomAdDImage",
            "image": "test-file-stub",
            "moreDescription": "Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident.",
            "title": "Florensis",
          }
        }
      />
    </div>
  </div>
</Fragment>
`;
